src/pages/media.astro

```diff
@@ -0,0 +1,504 @@
+---
+import BaseLayout from "../layouts/BaseLayout.astro";
+import Section from "../components/Section.astro";
+---
+
+<BaseLayout title="Media Kit">
+  <img
+    class="header-image"
+    src={"/img/banners/default-with-bear.svg"}
+    aria-hidden
+  />
+  <main id="main">
+    <Section variant="light" content="list">
+      <h1 class="align-center">JSHeroes Media Kit <br /> Welcome!</h1>
+
+      <div class="container column">
+        <h3>On this page you will find:</h3>
```

Comment:
this is not a heading semantically, it is just a paragraph text leading to the bullet list